### ISSUE TITLE
Error: Cannot find module 

### ISSUE THREAD

**Issue by steveszc** [ember-cli/ember-cli-clean-css#2]
After `ember install ember-cli-clean-css`, running `ember build -prod` throws the following error:
```
Error: Cannot find module '/my-ember-app/node_modules/ember-cli-clean-css:preprocessor'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at requireLocal (/my-ember-app/node_modules/process-relative-require/index.js:7:10)
    at module.exports.preprocessMinifyCss (/my-ember-app/node_modules/ember-cli-preprocess-registry/preprocessors.js:119:10)
    at EmberApp.styles (/my-ember-app/node_modules/ember-cli/lib/broccoli/ember-app.js:1356:30)
    at EmberApp.toArray (/my-ember-app/node_modules/ember-cli/lib/broccoli/ember-app.js:1663:12)
    at EmberApp.toTree (/my-ember-app/node_modules/ember-cli/lib/broccoli/ember-app.js:1684:38)
    at module.exports (/my-ember-app/ember-cli-build.js:112:14)
```

Running `ember build` finishes normally with no errors.

**Comment by ssavitski** [comment]
It's because of CSS minification, switch if off for prod environment and there'll be no such error.

**Comment by Turbo87** [comment]
this should have been fixed by https://github.com/ember-cli/ember-cli-preprocess-registry/pull/57, which was released as `ember-cli-preprocess-registry@3.2.2`
